vnext/Folly/windows_portability_tinyimp.cpp

```diff
@@ -0,0 +1,84 @@
+#include <folly/portability/PThread.h>
+
+#include <folly/portability/Sched.h>
+#include <folly/portability/Time.h>
+#include <folly/portability/Windows.h>
+
+namespace folly {
+namespace portability {
+namespace pthread {
+int pthread_key_create(pthread_key_t *key, void (*destructor)(void *)) {
+  DWORD newKey = FlsAlloc(nullptr);
+
+#pragma warning(push)
+#pragma warning(disable : 4312)
+  *key = reinterpret_cast<pthread_key_t>(newKey);
+#pragma warning(pop)
+
+  return 0;
+}
+
+int pthread_key_delete(pthread_key_t key) {
+#pragma warning(push)
+#pragma warning(disable : 4311)
+#pragma warning(disable : 4302)
```

Comment:
4302 is in the list of SDL warnings that must be fixed

Reply:
This code is no longer needed.